writings-volume-1/updated/02-chapter-1.adoc

```diff
@@ -1,6 +1,5 @@
 == Chapter I. Memoir of Isaac Penington
 
-[.blurb]
```

Comment:
I think we should put back the `[.blurb]` tag here. This is kind of a strange case, where there is a chapter title, and then a secondary heading/explanation under it that I think looks best centered and _italicised_... which the `[.blurb]` tag creates for us. Normally we don't use these `[.blurb]` tags right under a chapter heading, but occasionally (like here) I think it fits.

Reply:
I'm not sure why I did that....there's times when I'm erasing asterisms and I hold the delete button too long (I mostly try to highlight and delete now) and it's erased some other text unintentionally and I have to look in the other editions to see what I erased, or look in the originals. My apologies on this one, don't know why that happened!

Reply:
And part of the reason I didn't always highlight and delete is because my mouse didn't work the greatest and I couldn't move it very precisely but I got a new mousepad and that has helped. :)